src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -1275,7 +1284,13 @@ object FunctionRegistry {
     "s" -> "The string to parse."
   )(stringHr, locusIntervalHr(GR))
 
-  register("Interval", (chr: String, start: Int, end: Int) => Interval(Locus(chr, start), Locus(chr, end)),
+  registerDependent("Interval", () => {
+    val gr = GR.gr
+    (chr: String, start: Int, end: Int) => {
+      implicit val locusOrdering = gr.locusOrdering
```

Comment:
and here, I think? I guess you have to pass it twice so maybe it's worth it here.